provisioning/production/resources.tf

```diff
@@ -15,9 +15,4 @@ module "base" {
   source = "../base"
 
   environment = "production"
-
-  vpc_config = {
-    subnet_ids         = ["subnet-21a3b244", "subnet-f35de0a9"]
-    security_group_ids = ["sg-aa74f1db"]
-  }
```

Comment:
Is removing vpc supposed to be part of this?

Reply:
the commit is from a while ago, and it looks like it was on purpose. i'm guessing this is just an artifact of some sloppy git practice on my part. @nonword can you confirm that this app doesn't need vpc?

Reply:
Confirmed. We only need to run code in a VPC when connecting to locked down resources like databases and some Sierra services. This app connects only to public resources.